### DISCUSSION TITLE
Saving settings

### DISCUSSION THREAD

**Discussion by DamianMorozov** [DamianMorozov/OpenTgResearcher#6] [Ideas]
What about the new feature?
One of the settings is the path to the local database.

**Comment by DamianMorozov** [comment]
It's completed.

**Comment by ShiroUwUs7** [comment]
adding an option to 1. save the session id with the api hash and phone number (after agreeing to a warning etc.)
2. save the already imported list and just make it a number 1/2/3/4/5 etc. to pick the channel/sorce ID that was saved. 

so you dont have to enter in the hash id, phone number and group id whenever you re launch the application.

**Reply by DamianMorozov** [reply]
This will be fixed in the next version. But these settings will be stored in `Storage.db`.
